quickstart/armv7a9-zynq7000-zedboard.md

```diff
@@ -1,23 +1,43 @@
 # Running system on `armv7a9-zynq7000-zedboard` (Xilinx Zynq-7000, Zedboard development board)
 
 These instructions describe how to run a Phoenix-RTOS system image for `armv7a9-zynq7000-zedboard` target architecture.
-Note that, the build artifacts, including the system image, should be first provided in the `_boot` directory. If you haven't run the `build.sh` script yet, run it for `armv7a9-zynq7000-zedboard` target.
+Note that, the build artifacts, including the system image, should be first provided in the `_boot` directory.
+If you haven't run the `build.sh` script yet, run it for `armv7a9-zynq7000-zedboard` target.
 
 See [how to build the Phoenix-RTOS system image](../building/README.md).
 
 ## Preparing the board
 
-- Firstly, you should copy the disk image `phoenix-armv7a9-zynq7000-zedboard.disk` from the `_boot` directory to the SD card and rename it to `BOOT.bin`.
+Preparing the board depends on how the plo is loaded into RAM, this quickstart describes 2 approaches.
+Loading from SD card and from NOR flash, depending on your needs use one of them.
+For example if you are flashing Phoenix-RTOS for a first time, or you want to load a plo from the current system image.
+Otherwise, you can simply use plo from the already flashed image.
+
+### Loading plo form SD card
+
+- Firstly, you should copy the disk image `phoenix.disk`
+ from the `_boot/armv7a9-zynq7000-zedboard` directory to the SD card and rename it to `BOOT.bin`.
 
 - Then, insert the SD card into the board.
 
-- To allow booting from SD card, set the jumpers to the following configuration:
+- To allow booting from SD card, set the jumpers to
+ the following configuration (`JP7`: `110`, `JP8`: `110`, `JP9`: `011`, `JP10`: `011`, `JP11`: `110`):
 
   <img src="_images/zynq7000-sd-boot-mode.jpg" width="400px">
 
-- After that, to provide a power supply for the board, you should connect AC Adapter to the DC socket on the board. For now, leave the `SW8` switch in the `OFF` position.
+### Loading plo from NOR flash
```

Comment:
IMHO it would be unclear, because quickstart is applicable not only for the users, who use system on the specific board for a first time. That's why I would prefer to present two scenarios from the beginning, but it's a good remark, I'd like to leave the following note instead:

**This version is possible only if you have already flashed Phoenix-RTOS system image to this board before!**

Reply:
I love this idea